src/SDKs/ManagedServices/ManagedServices.Test/ManagedServices.Test.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.test.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  <PropertyGroup>
+    <PackageId>Foo</PackageId>
```

Comment:
Please fix the name and other tags
AssemblyVersion must be set to 1.0..0 and never changed for test projects

Reply:
Done